fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexer.java

```diff
@@ -991,13 +1004,26 @@ public Builder setIncreaseLimitAfter(int increaseLimitAfter) {
                 return this;
             }
 
+            /**
+             * Set the time limit. The indexer will exit with a proper exception if this time is exceeded after a non-final
```

Comment:
Same comment about the word "iteration" as used above.

Reply:
Changed.